demos/link-prediction/knowledge-graphs/distmult.ipynb

```diff
@@ -0,0 +1,1818 @@
+      "Epoch 286/300\n",
+      "11/11 [==============================] - 1s 70ms/step - loss: 0.0468 - binary_accuracy: 0.9993 - val_loss: 0.1455 - val_binary_accuracy: 0.9799\n",
+      "Epoch 287/300\n",
+      "11/11 [==============================] - 1s 70ms/step - loss: 0.0461 - binary_accuracy: 0.9994 - val_loss: 0.1450 - val_binary_accuracy: 0.9801\n",
+      "Epoch 288/300\n",
+      "11/11 [==============================] - 1s 68ms/step - loss: 0.0459 - binary_accuracy: 0.9993 - val_loss: 0.1451 - val_binary_accuracy: 0.9804\n",
+      "Epoch 289/300\n",
+      "11/11 [==============================] - 1s 69ms/step - loss: 0.0469 - binary_accuracy: 0.9994 - val_loss: 0.1435 - val_binary_accuracy: 0.9806\n",
+      "Epoch 290/300\n",
+      "11/11 [==============================] - 1s 70ms/step - loss: 0.0458 - binary_accuracy: 0.9994 - val_loss: 0.1436 - val_binary_accuracy: 0.9806\n",
+      "Epoch 291/300\n",
+      "11/11 [==============================] - 1s 70ms/step - loss: 0.0462 - binary_accuracy: 0.9994 - val_loss: 0.1440 - val_binary_accuracy: 0.9803\n",
+      "Epoch 292/300\n",
+      "11/11 [==============================] - 1s 67ms/step - loss: 0.0463 - binary_accuracy: 0.9994 - val_loss: 0.1438 - val_binary_accuracy: 0.9806\n",
+      "Epoch 293/300\n",
+      "11/11 [==============================] - 1s 70ms/step - loss: 0.0455 - binary_accuracy: 0.9994 - val_loss: 0.1452 - val_binary_accuracy: 0.9800\n",
+      "Epoch 294/300\n",
+      "11/11 [==============================] - 1s 67ms/step - loss: 0.0460 - binary_accuracy: 0.9993 - val_loss: 0.1455 - val_binary_accuracy: 0.9803\n",
+      "Epoch 295/300\n",
+      "11/11 [==============================] - 1s 69ms/step - loss: 0.0459 - binary_accuracy: 0.9993 - val_loss: 0.1440 - val_binary_accuracy: 0.9803\n",
+      "Epoch 296/300\n",
+      "11/11 [==============================] - 1s 68ms/step - loss: 0.0458 - binary_accuracy: 0.9993 - val_loss: 0.1443 - val_binary_accuracy: 0.9805\n",
+      "Epoch 297/300\n",
+      "11/11 [==============================] - 1s 69ms/step - loss: 0.0456 - binary_accuracy: 0.9994 - val_loss: 0.1444 - val_binary_accuracy: 0.9803\n",
+      "Epoch 298/300\n",
+      "11/11 [==============================] - 1s 68ms/step - loss: 0.0457 - binary_accuracy: 0.9993 - val_loss: 0.1459 - val_binary_accuracy: 0.9803\n",
+      "Epoch 299/300\n",
+      "11/11 [==============================] - 1s 69ms/step - loss: 0.0449 - binary_accuracy: 0.9994 - val_loss: 0.1438 - val_binary_accuracy: 0.9806\n",
+      "Epoch 300/300\n",
+      "11/11 [==============================] - 1s 69ms/step - loss: 0.0446 - binary_accuracy: 0.9994 - val_loss: 0.1456 - val_binary_accuracy: 0.9805\n"
+     ]
+    }
+   ],
+   "source": [
+    "wn18_es = callbacks.EarlyStopping(monitor=\"val_loss\", patience=50)\n",
+    "wn18_history = wn18_model.fit(\n",
+    "    wn18_train_gen, validation_data=wn18_valid_gen, epochs=epochs, callbacks=[wn18_es]\n",
```

Comment:
You train for a fair few epochs here! I think it would improve readability if we set `verbose=0`. And the same below for FB15k.

```suggestion
    "    wn18_train_gen, validation_data=wn18_valid_gen, epochs=epochs, callbacks=[wn18_es], verbose=0\n",
```